front-end/src/components/Post/Poll/votes.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+export const AYE = 'AYE';
```

Comment:
We should use an enum with types, the same way we do elsewhere.

Reply:
Done